src/display/api.js

```diff
@@ -455,6 +479,8 @@ function _fetchDocument(worker, source, pdfDataRangeTransport, docId) {
       isEvalSupported: source.isEvalSupported,
       fontExtraProperties: source.fontExtraProperties,
       enableXfa: source.enableXfa,
+      standardFontDataUrl: source.standardFontDataUrl,
+      standardFontDataWorkerFetch: source.standardFontDataWorkerFetch,
```

Comment:
As mentioned in https://github.com/mozilla/pdf.js/pull/12726/files#r634076823, please change this to the following instead to remove the need to send *two* very closely related properties:
```suggestion
      standardFontDataUrl:
        source.standardFontDataWorkerFetch ? source.standardFontDataUrl : null,
```